guide/plg_atTheTelescope.tex

```diff
@@ -1153,7 +1156,189 @@ \subsection*{Author}
 \label{sec:plugins:Observability:author}
 This plugin has been contributed by Ivan Marti-Vidal (Onsala Space Observatory)\footnote{\url{mailto:i.martividal@gmail.com}} with some advice by Alexander Wolf and Georg Zotti.
 
+\newpage
+\section{Mosaic Camera Plugin}
+\sectionauthor*{Josh Meyers\footnote{\url{mailto:jmeyers314@gmail.com}}}
+\label{sec:plugins:MosaicCamera}
+
+\begin{figure}[t]\centering
+\includegraphics[width=\textwidth]{MosaicCamera.png}
+\caption{The Mosaic Camera plugin in action.  Overlays of LSSTCam(left), DECam (center), and HyperSuprimeCam (right) are shown, as well as the configuration dialog for the plugin (top right).}
+\label{fig:plugins:MosaicCamera}
+\end{figure}
+
+\subsection{Introduction}
+\label{sec:plugins:MosaicCamera:intro}
+This plugin overlays an outline of one or more mosaic cameras, i.e., cameras with multiple sensors, onto the Stellarium celestial sphere.
+With this functionality, one may compare the fields of view of different cameras, including potentially one's own astrophotography setup.
+Combined with the scripting and remote control features of Stellarium, various visualizations are possible, including real-time tracking of the observations of telescopes, playback of past observations, or public outreach demonstrations of the capabilities of various instruments.
+
+\subsection{Usage}
+\label{sec:plugins:MosaicCamera:usage}
+
+To use the plugin, first enable it by selecting the "Load at startup" checkbox under the "Plugins" tab of the main Configuration window.
```

Comment:
Please use "grave accents" to start a quotation, and apostrophes to end. Like  ` ``Plugins'' `
Also, IMHO the first sentence is not needed. General plugin handling is known or described elsewhere.